docs/BUG-BOUNTY.md

```diff
@@ -0,0 +1,78 @@
+# The curl bug bounty
+
+ The curl project runs a bug bounty program in association with
+ bountygraph.com.
+
+ After you have reported a security issue to the curl project, it has been
+ deemed credible and a patch and advisory has been made public you can be
+ eligible for a bounty from this program.
+
+ See all details at https://bountygraph.com/programs/curl
+
+ This bounty is relying on funds from sponsors. If you use curl professionally,
+ consider help funding this!
+
+## How much money is the bounty at
+
+ The curl projects offer monetary compensation for reported and published
+ security vulnerabilities. The amount of money rewarded depends on how serious
+ the flaw is determined to be.
+
+ We offer reward money *up to* these amounts. The curl security team will
+ solely and exclusively determine the exact amount for each reported flaw on a
+ case by case basis and keep the rights to adjust the amount as it sees fit.
+
+ - Low         $500
+ - Medium    $1,000
+ - High      $5,000
+ - Critical $10,000
```

Comment:
Nitpick, but should we use "USD 10,000" instead as $ is the denomination for many currencies (living in Australia taught me the confusions that can be had).

Reply:
Makes sense yes, I'll amend.